test/extended/security/scc.go

```diff
@@ -263,6 +279,188 @@ var _ = g.Describe("[sig-auth][Feature:SecurityContextConstraints] ", func() {
 			t.Fatalf("user2 failed PSP SSR in project2: %v", allowedBy)
 		}
 	})
+
+	g.It("TestAllowedSCCViaRBAC with service account [apigroup:security.openshift.io]", func() {
```

Comment:
this is again copy-pasta, see if you can abstract the logic in a function again

Reply:
Updated